### PR TITLE
Drawer image

### PR DESCRIPTION
**Pull request by jakedcolbert**

### CODE REVIEW

File: vite.config.js.timestamp-1667841673769.mjs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// vite.config.js
```
Comment (chrisvire) [inline vite.config.js.timestamp-1667841673769.mjs:1]: What is this file?  Is it a generated temporary file? Does it need .gitignore'd?

File: src/lib/components/Sidebar.svelte [hunk 2]
```diff
@@ -45,7 +45,10 @@ The sidebar/drawer.
         <label for={drawerId} class="dy-drawer-overlay" />
         <ul class="dy-menu p-1 overflow-y-auto w-3/4 sm:w-80 bg-base-100 text-base-content">
             <!-- Sidebar content here -->
-            <li><a href="/"><BibleIcon /><span>{$globalConfig.name}</span></a></li>
+            <a class="fill" href="/">
```
Comment (chrisvire) [inline src/lib/components/Sidebar.svelte:48]: What happens if there is no nav_drawer.png or nav_drawer@2x.png specified?

**Comment by chrisvire** [comment]
Comparing screenshot from Android and Chrome when in phone simulation mode.

Chrome (Pixel 5)

image

Android (Pixel 3XL)

image

It feels a little wide, but it is probably fine.

File: src/lib/components/Sidebar.svelte (hunk 2 above)
Comment (chrisvire) [reply src/lib/components/Sidebar.svelte:48]: This is what happens:

image

**Review by chrisvire — CHANGES_REQUESTED** [review]
1) Please handle not having a Navigation Drawer image.  The Android App uses a default app icon.  You could use the Icon that was there before?

2) Figure out where the vite.config.js.timestamp-XXXX.mjs came from and stop it being generated (or at a minimum .gitignore files like it -- would need a file pattern).

File: src/lib/components/Sidebar.svelte (hunk 2 above)
Comment (chrisvire) [reply src/lib/components/Sidebar.svelte:48]: I like having an icon and the App Name if there is no Drawer Image.  Do you want me to try to get the App Icon or just use the icon we were using before?

File: vite.config.js.timestamp-1667841673769.mjs (hunk 1 above)
Comment (jakedcolbert) [reply vite.config.js.timestamp-1667841673769.mjs:1]: looks like it came from the npm run format

File: src/lib/components/Sidebar.svelte (hunk 2 above)
Comment (chrisvire) [reply src/lib/components/Sidebar.svelte:48]: I talked it over with another developer and he said the back end should make sure there is a file.  So this is OK.

File: vite.config.js.timestamp-1667841673769.mjs (hunk 1 above)
Comment (chrisvire) [reply vite.config.js.timestamp-1667841673769.mjs:1]: So remove this from the PR and it should be good to go.

File: src/lib/components/Sidebar.svelte [hunk 3]
```diff
@@ -39,13 +39,18 @@ The sidebar/drawer.
     <input id={drawerId} type="checkbox" class="dy-drawer-toggle" />
     <div class="dy-drawer-content flex flex-col bg-gray-200">
         <!-- Page content here -->
-        <slot />
+        <slot d />
```
Comment (chrisvire) [inline src/lib/components/Sidebar.svelte:42]: Why is this here?
